docs/source/variables.ipynb

```diff
@@ -6,7 +6,7 @@
    "source": [
     "# Variable Types\n",
     "\n",
-    "A Variable is analogous to a column in table in a relational database. When running DFS, Featuretools will attempt to infer the types of variable present in your data. DFS also allows for specifying the variable types.\n",
+    "A Variable is analogous to a column in a table in a relational database. When running DFS, Featuretools will attempt to infer the types of variables present in each Entity of the EntitySet. DFS also allows for specifying the variable types.\n",
```

Comment:
It depends on how technical we want to get here, but inferring variable types happens before running DFS.

Reply:
Not sure if it is worth including or not, but we don't currently perform variable type inference for Dask Entities. Users must specify the variable types for all the columns in the entity dataframe.

As a side note - this document will be helpful in helping users pick the right type to use.

Reply:
Fixed.
I changed the wording to:

> When creating an Entity, Featuretools will attempt to infer the types of variables present. Featuretools also allows for explicitly specifying the variable types when creating the Entity."


I put a note:

>  Note: When using Dask Entities, users must explicitly specify the variable types for all columns in the Entity dataframe.